Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3.11.11-slim
+
+ENV PYTHONBUFFERED 1
```

Comment:
There is a typo in the `ENV` instruction. It should be `PYTHONUNBUFFERED` instead of `PYTHONBUFFERED` to ensure that Python output is sent straight to the terminal without being buffered.